### ISSUE TITLE
Too many actions needed to access an app when screen is off

### ISSUE THREAD

**Issue by lduboeuf** [ubports/unity8#217]
If i want to open an app from the Drawer while my phone still locked, i have to swipe right first to open the Lock prompt, log in and then swipe right from edge to open the Drawer. Found it a bit tricky.
In my opinion, when we wake-up the phone, we should be able to swipe only once when we want to access to the Drawer

**Comment by dobey** [comment]
I don't quite understand what you're asking for here. The phone can't intuit that you will want to open the app drawer when you're unlocking the phone. If you don't have any existing apps open though, I think the launcher should be shown by default in this case, and you can just tap the logo button at the bottom instead of swiping. Is that not the case? Can you more clearly explain what you are asking for here?

**Comment by lduboeuf** [comment]
humm so the use case: I wake up the screen by pressing the power button, now if i drag far right from the edge, only the Launcher is displayed, i cannot open the Drawer. I need to start dragging out of the Launcher area in order to unlock the phone and then drag again from the edge to display the app list or by pressing the Home logo. Found it a bit tricky

**Comment by dobey** [comment]
Yes, you need to log in to launch apps. Really, I would say the launcher should not be shown on lock screen, for privacy reasons. This was the same with scopes. You can't get to the apps scope on the lock screen, only the launcher panel.

**Comment by lduboeuf** [comment]
yes of course, better not to show the Drawer if not logged in but at least having the greeter displayed to avoid unecessary swipe actions ?

**Comment by dobey** [comment]
I don't understand what you mean here. How is the greeter not displayed? Can you post a screenshot? To me it sounds like the greeter is displayed, but you're trying to open the drawer.

**Comment by lduboeuf** [comment]
if you drag far right from edge when not logged in, the greeter is not displayed, only the Launcher. You see ?

**Comment by cibersheep** [comment]
- You can prevent launcher and/or indicators bar to be accessible from lock screen from System Settings > Privacy > Lock / unlock
- The use case you propose here, I would say is very specific. Now Lomiri is behaving (as design) like Unity. I wouldn't force the dash to show when you unlock the device to be honest.

**Comment by dobey** [comment]
> if you drag far right from edge when not logged in, the greeter is not displayed, only the Launcher. You see ?

Oh. Just tried this again, and I think I understand now. The greeter is displayed, but the "swipe to unlock" is not moved, when swiping from the left edge. The launcher remains instead. I'm not sure this specifically is an issue, though there may be a separate issue open already about launcher being accessible on lock screen by default.

**Comment by lduboeuf** [comment]
Yes i think it's just a matter of habit. I close this as not very relevant, and sorry for not being clear, i confused Greeter with the login prompt
